step.yml

```diff
@@ -1,4 +1,4 @@
-title: Deploy to iTunes Connect with Deliver
+title: Deploy to App Store Connect with Deliver
```

Comment:
Should we include `(iTunes Connect)`, so the step can be found using the search in workflow editor?

Reply:
Good idea, fixed